include/ginkgo/core/log/papi.hpp

```diff
@@ -265,12 +273,10 @@ class Papi : public Logger {
 
         ~papi_queue()
         {
-            if (PAPI_is_initialized()) {
```

Comment:
is it always initialized now?

Reply:
Here we don't rely on `PAPI` anymore. We only rely on the `sde_lib` (see headers). There is no initialization or deinitialization anymare. It only happens in the tests or user application side.